conda-store-server/tests/plugins/test_plugin_context.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) conda-store development team. All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+import logging
+import os
+import subprocess
+
+import pytest
+
+from conda_store_server.plugins.plugin_context import PluginContext
+
+
+class TestOutPutCapture:
+    def __init__(self):
+        self.output = ""
+
+    def write(self, line: str):
+        self.output += line
+
+
+def test_run_command_no_logs():
+    out = TestOutPutCapture()
+    err = TestOutPutCapture()
+    pr = PluginContext(stdout=out, stderr=err, log_level=logging.ERROR)
+
+    pr.run_command(["echo", "testing"])
+    assert err.output == ""
+    assert out.output == "testing\n"
+
+
+def test_run_command_log_info():
+    out = TestOutPutCapture()
+    err = TestOutPutCapture()
+    pr = PluginContext(stdout=out, stderr=err, log_level=logging.INFO)
+
+    pr.run_command(["echo", "testing"])
+    assert err.output == ""
+    assert (
+        out.output
+        == """Running command: ['echo', 'testing']
+testing
+"""
+    )
+
+
+def test_run_command_errors():
+    out = TestOutPutCapture()
+    err = TestOutPutCapture()
+    pr = PluginContext(stdout=out, stderr=err, log_level=logging.ERROR)
```

Comment:
Why `pr` for an instance of `PluginContext`? :thinking: `plugin_context` or `context` should work ok.